### PR TITLE
[wtl] Fix #includes in wtl/atlribbon.h

### LINKED ISSUE #24933: [wtl] Error when including atlribbon.h
****

I'm trying to use the wtl package, but as soon as I `#include <wtl/atlribbon.h>`, the build fails:

    ...\vcpkg_installed\x86-windows\x86-windows\include\wtl\atlribbon.h(30,10): fatal error C1083: Cannot open include file: 'atlmisc.h': No such file or directory

The error comes from the atlribbon.h file in the WTL package, and from the fact that vcpkg sets the include path to the folder containing the "wtl" folder, not the folder itself. Therefore, I also had to change my includes from `#include <atlribbon.h>` to `#include <wtl/atlribbon.h>` in the first place. WTL just assumes that its include folder is directly in the path.

Easy steps to reproduce:
- Install the WTL package into VS 2019 (I didn't try VS 2022)
- create a new WTL project, any type
- add a vcpkg.json that references the "wtl" package
- switch on "Use Vcpkg Manifest"
- compile

**Proposed solution**

The solution would be to change the include path for the WTL package to directly include the "wtl" folder in the package, not the parent folder of that. After all, the WTL project wizard creates projects with this in mind and surely there are WTL projects out there who use that convention for decade(s). I'm not sure if this breaks some vcpkg rules. At the moment, the package is useless.

**Describe alternatives you've considered**

I can manually add an "Additional Include Directory" in the Visual Studio C/C++ properties, as a workaround:

    $(VcpkgManifestRoot)\vcpkg_installed\$(VcpkgTriplet)\$(VcpkgTriplet)\include\wtl

Note that this is also needed to be added to the Resource Compiler settings, as the .rc files may also include <atlres.h>, which is also a part of WTL.

### PR DESCRIPTION
**Pull request by chrullrich**
- #### What does your PR fix?  
  Fixes #24933.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  No. What?

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  I'd say yes, but why would you trust me?

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  No. I *have* run `vcpkg x-add-version wtl`, though; I hope that suffices.

### CODE REVIEW

File: ports/wtl/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "wtl",
-  "version-string": "10.0.10320",
-  "port-version": 2,
+  "version": "10.0.10320.0",
+  "port-version": 3,
```
Comment (Adela0814) [inline ports/wtl/vcpkg.json:4]: The `port-version` should be removed if the `version `has changed.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: versions/w-/wtl.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "ba8497fd0c3195e5557dcb1e886de08d03172788",
+      "version-string": "10.0.10320",
+      "port-version": 3
+    },
+    {
+      "git-tree": "e3f35cab3e5b87f23f41cf4e4d90d2170257f9e0",
+      "version": "10.0.10320.0",
+      "port-version": 3
+    },
```
Comment (Thomas1664) [inline versions/w-/wtl.json:12]:
```suggestion
```

File: ports/wtl/vcpkg.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "wtl",
   "version-string": "10.0.10320",
-  "port-version": 2,
+  "port-version": 3,
```
Comment (Thomas1664) [inline ports/wtl/vcpkg.json:4]: Please change `version-string` to `version`

**Comment by chrullrich** [comment]
OK, never mind. I tried.

**Pull request closed without merging**